data/korath/korath missions.txt

```diff
@@ -257,3 +257,179 @@ mission "Small Scale Delivery"
+					goto defenses
+				`	"Afterburners and fuel, of course."`
+					goto afterburners
+				`	"Power and cooling would be wonderful."`
+					goto coolpower
+				`	"I'll take the pair of fiery turrets."`
+					goto inferno
+				`	"Doomsday torpedoes? Count me in!"`
+					goto torpedoes
+
+			# This is the default option because you can't go wrong with defenses.
+			# A mix of outfit types and sizes lets you outfit a wide variety of
+			# designs. It's easy to pillage any of these, so this is the wimpiest
+			# option, but also the most versatile.
+			label defenses
+			action
+				outfit "Systems Core (Medium)"
+				outfit "Systems Core (Small)"
+				outfit "Warder Anti-Missile" 2
+				outfit "Small Heat Shunt"
+				outfit "Large Heat Shunt"
+				outfit "Fuel Processor"
+			`	"Done! We're also giving you <payment>. These outfits keep you alive through tough fights."`
+				goto end
+
+			# You need serious power and cooling to support two Infernos,
+			# so the gift is smaller than it appears.
+			label inferno
+			action
+				outfit "Korath Inferno" 2
+			`	"Done! We're also giving you <payment>. You'll need some hefty cooling and a lot of weapon space to support these turrets."`
+				goto end
+
+			# This much power and cooling opens up a lot of designs if you
+			# have a Shield Beetle, but it isn't so useful on human ships.
+			label coolpower
+			action
+				outfit "Triple Plasma Core"
+				outfit "Liquid Sodium Cooler"
+				outfit "Large Heat Shunt"
+			`	"Done! We're also giving you <payment>. You'll need a Shield Beetle to go with this generator, as nothing else from humans or Hai has enough space to make effective use of it."`
+				goto end
+
+			# Do you want a capital ship that can outrun drones?
+			label afterburners
+			action
+				outfit "Afterburner (Planetary Class)"
+				outfit "Fuel Processor" 3
+			`	"Done! We're also giving you <payment>. Make sure you have enough cooling and engine space to support this afterburner."`
+				goto end
+
+			# This is a secret for players to find. It is the only time
+			# in the game where you can get Firestorm Torpedo Racks.
+			# Anything else you can pillage, so by some measure this is
+			# the "right" option.
+			label torpedoes
+			action
+				outfit "Firestorm Torpedo Rack" 2
+				outfit "Firestorm Torpedo" 39
+				outfit "Firestorm Battery"
+			`	"Done! We're also giving you <payment>. Be careful with these torpedoes. They have a wide blast radius. They'll cripple your ship and escorts if they detonate too close."`
+				goto end
+
+			label end
+			`	"It's always a pleasure to help another warrior get their due."`
+
+	npc kill
+		government Korath
+		system Almach
+		personality staying timid target marked
+		fleet "Korath Stealth Relay"
+		dialog "With all of the probes destroyed, you can now return to Stormhold for your payment."
+	npc
+		government Korath
+		system Almach
+		personality staying heroic marked opportunistic unconstrained
+		fleet
+			variant
+				"Ra'gru Ik 618"
```

Comment:
I didnt just ask to make it capturable, I meant a capturable and fully crewed ship (look at the Ground Assault variant that already exists for another Korath ship)

Reply:
There is now a `Ra'gru Ik 618 (Ground Assault)` variant and it is used for the mission.

Reply:
The default 618 already has enough rifles for all of its bunks.

Reply:
great, this small change should make it a bit harder to capture this guarantied to spawn ship